packages/model/src/reminder.ts

```diff
@@ -50,6 +54,20 @@ export interface ResearchOutputPublishedReminder
   };
 }
 
+export interface ResearchOutputDraftReminder extends ResearchOutputReminder {
+  entity: 'Research Output';
+  type: 'Draft';
+  data: {
+    researchOutputId: ResearchOutputDraftDataObject['id'];
+    documentType: ResearchOutputDraftDataObject['documentType'];
```

Comment:
I believe the `documentType` and `addedDate` are not necessary 😉

Reply:
addedDate is necessary since we sort the reminders by that addedDate, documentType indeed is not